### PR TITLE
[ZT] Expand DLP documentation

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+pcx-content-type: how-to
+title: Data Loss Prevention
+weight: 5
+layout: single
+---
+
+# Data Loss Prevention (beta)
+
+With Cloudflare Data Loss Prevention (DLP) and Secure Web Gateway, you can inspect HTTP traffic for the presence of sensitive data such as social security numbers and credit card numbers. DLP scans the entire HTTP body, which may include uploaded or downloaded Microsoft Office documents, chat messages, forms, and other web content. Visibility varies depending on the site or application. DLP does not scan non-HTTP traffic such as email, nor does it scan any traffic that bypasses Cloudflare Gateway (for example, traffic that matches a [_Do Not Inspect_](/cloudflare-one/policies/filtering/http-policies/#do-not-inspect) rule).
```
Comment (cloudflare-noelle) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]: Can we update "Microsoft Office documents" to "Microsoft Office documents (Office 2007 and later)"

**Review by cloudflare-noelle — CHANGES_REQUESTED** [review]
Added one comment for an update I realized we also needed.  Looks great! Nice Work!

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md (hunk 1 above)
Comment (ranbel) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md]:
```suggestion
With Cloudflare Data Loss Prevention (DLP) and Secure Web Gateway, you can inspect HTTP traffic for the presence of sensitive data such as social security numbers and credit card numbers. DLP scans the entire HTTP body, which may include uploaded or downloaded Microsoft Office documents (Office 2007 and later), chat messages, forms, and other web content. Visibility varies depending on the site or application. DLP does not scan non-HTTP traffic such as email, nor does it scan any traffic that bypasses Cloudflare Gateway (for example, traffic that matches a [_Do Not Inspect_](/cloudflare-one/policies/filtering/http-policies/#do-not-inspect) rule).
```

**Review by cloudflare-noelle — APPROVED** [review]

File: content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+layout: single
+---
+
+# Data Loss Prevention (beta)
+
+With Cloudflare Data Loss Prevention (DLP) and Secure Web Gateway, you can inspect HTTP traffic for the presence of sensitive data such as social security numbers and credit card numbers. DLP scans the entire HTTP body, which may include uploaded or downloaded Microsoft Office documents (Office 2007 and later), chat messages, forms, and other web content. Visibility varies depending on the site or application. DLP does not scan non-HTTP traffic such as email, nor does it scan any traffic that bypasses Cloudflare Gateway (for example, traffic that matches a [_Do Not Inspect_](/cloudflare-one/policies/filtering/http-policies/#do-not-inspect) rule).
+
+To perform DLP filtering, first configure a DLP Profile with the data patterns you want to detect, and then build a Gateway HTTP policy to allow or block the sensitive data from leaving your organization. Gateway will parse and scan your HTTP traffic for strings matching the keywords or regexes specified in the DLP profile.
+
+## Prerequisites
```
Comment (kodster28) [inline content/cloudflare-one/policies/filtering/http-policies/data-loss-prevention/_index.md:14]: Personally, might recommend breaking this out onto a subpage, but not a super strong preference here.

**Review by kodster28 — APPROVED** [review]

**Pull request merged**